tools/cldr-code/src/test/java/org/unicode/cldr/util/TestCoverageLevel2.java

```diff
@@ -44,7 +46,6 @@ public void TestCoveragePerf() {
     @Test
     public void TestPriorBasicLanguage() throws IOException {
```

Comment:
Everything is aggregated - so that it will print out ALL errors and not just stop at the first.